packages/cli/src/cli/index.ts

```diff
@@ -198,6 +197,108 @@ $$$$$$\\   $$$$$$$\\  $$\\  $$$$$$\\   $$$$$$$ |$$\\  $$\\  $$\\  $$$$$$\\  $$$$
       await detectExtensions(options);
     });
 
+  // eslint-disable-next-line turbo/no-undeclared-env-vars
+  if (process.env.THIRDWEB_CLI_SKIP_INTRO !== "true") {
```

Comment:
since ur looking this up multiple times might make sense to extract to const up top

Reply:
good idea